packages/checkbox/src/react/index.js

```diff
@@ -68,7 +81,8 @@ const Checkbox = React.forwardRef((props, ref) => {
           tabIndex={props.disabled ? '-1' : '0'}
           {...styles.square(themeName, props)}
         >
-          {props.checked && <Checkmark />}
+          {props.indeterminate && <Indeterminate />}
+          {props.checked && !props.indeterminate && <Checkmark />}
```

Comment:
Kind of an odd api, but apparently it matches the native checkbox api. Who knew!  https://css-tricks.com/indeterminate-checkboxes/